### PR TITLE
gui: Some improve for gui

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- gui/manual: Improve D-pad in slider
- gui/live area: Remove the zoom instructions
- gui/settings dialog: Add lang to main buttons

### CODE REVIEW

File: vita3k/gui/src/app_context_menu.cpp [hunk 1]
```diff
@@ -559,8 +559,12 @@ void draw_app_context_menu(GuiState &gui, EmuEnvState &emuenv, const std::string
             ImGui::SetCursorPosX((WINDOW_SIZE.x / 2.f) + (20.f * SCALE.x));
         }
         if (ImGui::Button(common["ok"].c_str(), BUTTON_SIZE) || ImGui::IsKeyPressed(emuenv.cfg.keyboard_button_cross)) {
-            if (context_dialog == lang["app_delete"])
+            if (context_dialog == lang["app_delete"]) {
                 delete_app(gui, emuenv, app_path);
+                fs::remove_all(ADDCONT_PATH);
+                fs::remove_all(LICENSE_PATH);
+                fs::remove_all(SAVE_DATA_PATH);
+            }
```
Comment (Zangetsu38) [inline vita3k/gui/src/app_context_menu.cpp:567]: revert

File: lang/fr.xml [hunk 2]
```diff
@@ -423,6 +419,14 @@ Votre système Vita3K est prêt !</completed_setup>
...
 	</settings>
 
+	<settings_dialog>
+		<close>Close</close>
+		<save_reboot>Save & Reboot</save_reboot>
+		<save_apply>Save & Apply</save_apply>
+		<save>Sauvegarder</save>
+		<keep_changes>Click on Save to keep your changes.</keep_changes>
+	</settings_dialog>
+
```
Comment (Zangetsu38) [inline lang/fr.xml:429]: no set lang string if you keep it in english, just using global variable
same for all

File: vita3k/gui/src/app_context_menu.cpp (hunk 1 above)
Comment (Croden1999) [reply vita3k/gui/src/app_context_menu.cpp:567]: if delete applation, only remove app path, not delete save data and dlc, does not match the description

Comment (Zangetsu38) [reply vita3k/gui/src/app_context_menu.cpp:567]: I know what i said no worry, revert it 👌

**Comment by CoffeeBrewer64** [comment]
Crashes on Windows when clicking "Help" button
image

**Comment by Croden1999** [comment]
please Mr. Zangetsu check again

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**